### ISSUE TITLE
jax.ops has no attribute 'index_add'

### ISSUE THREAD

**Issue by AlenGhzau** [deepmind/alphafold#558]
warning!warning!warning!
the  latest version of jax has  removed `index_update` and `index_add`,
which caused the code in alphafold2/alphafold2/model/modules.py should be edited
the code in line 1725
` msa_activations = jax.ops.index_add(msa_activations, 0,prev_msa_first_row)`
should be changed!!!!!
i get this message from the explain of the author of jax:
https://github.com/google/jax/issues/11706

**Comment by joshabramson** [comment]
Thanks for reporting this issue. In general we advise pinning to the version of JAX defined in the requirements.txt, rather than installing any newer packages. This is because we do not check that newer version of JAX produce the same level of accuracy.

Regardless, these ops are not in the latest versions of the model.

**Comment by joshabramson** [comment]
Hi, thanks for reporting this issue. In general we advise pinning to the version of JAX defined in the requirements.txt, rather than installing any newer packages. This is because we do not check that newer version of JAX produce the same level of accuracy.

In any case, these ops are not in the newest versions of the models.
